webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/SystemProperties/index.js

```diff
@@ -0,0 +1,214 @@
+        <DescriptionListTerm>{__('Model')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          <SkeletonLoader
+            status={status}
+            emptyState={<DefaultLoaderEmptyState />}
+          >
+            {model}
+          </SkeletonLoader>
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Host group')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          <SkeletonLoader
+            status={status}
+            emptyState={<DefaultLoaderEmptyState />}
+          >
+            {hostgroupName}
+          </SkeletonLoader>
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Owner')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          <SkeletonLoader
+            status={status}
+            emptyState={<DefaultLoaderEmptyState />}
+          >
+            {ownerName}
+          </SkeletonLoader>
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Owner type')}</DescriptionListTerm>
+        <DescriptionListDescription>{ownerType}</DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Location')}</DescriptionListTerm>
+        <DescriptionListDescription>{location}</DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Organization')}</DescriptionListTerm>
+        <DescriptionListDescription>{organization}</DescriptionListDescription>
+      </DescriptionListGroup>
+      <Slot id="host-details-tab-properties-2" multi />
+    </DescriptionList>
+    <Divider className="padded-divider" />
+    <DescriptionList isCompact>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Created at')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {createdAt && <LongDateTime date={createdAt} />}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Updated at')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          <SkeletonLoader
+            status={status}
+            emptyState={<DefaultLoaderEmptyState />}
+          >
+            {updateAt && <LongDateTime date={updateAt} />}
+          </SkeletonLoader>
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Uptime (s)')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          <SkeletonLoader
+            status={status}
+            emptyState={<DefaultLoaderEmptyState />}
+          >
+            {/* render uptime in hh:mm:ss format */}
+            {uptime && new Date(uptime * 1000).toISOString().substr(11, 8)}
```

Comment:
Why do you do a substring of a date? Isn't it way better to use a proper date format? Also, how would it render if the uptime is more than a day? Uptimes of days or weeks are not uncommon.

Reply:
the `uptime` is stored in seconds, so what would be a better and less confusing format: 
`hh:mm:ss`
or
`dd:hh:mm:ss`

Reply:
I thought we normalized it to the boot time. For example, here's how we display it in the current UI.
https://github.com/theforeman/foreman/blob/57a4506c6060dbf34e6b94e322c2316b1d50a808/app/helpers/host_description_helper.rb#L76

Reply:
is the `boot_time` alias to `uptime_seconds` attribute?

Reply:
Not an alias since boot_time is the time the system booted up. This is constant for a system while it stays up. The uptime is the time that passed since booting. Here it's derived from the boot_time:
https://github.com/ekohl/foreman/blob/0fcb552e9c26dd4c31963324255f8910c9afda0c/app/models/host_facets/reported_data_facet.rb#L21-L23